### ISSUE TITLE
Call for mantainers!

### ISSUE THREAD

**Issue by mjrussell** [mjrussell/redux-auth-wrapper#315]
I'm currently unable to continue the maintenance of redux-auth-wrapper. While the lines of code and overall activity on this library is fairly small, there are several feature requests and people looking to use this library with the latest versions of history and react router.

Please let me know in the comments if you are interested in becoming a maintainer. Ideally you would start by implementing a feature or upgrading some dependencies as a first step.

**Comment by mjrussell** [comment]
@veeramarni is this something you or someone from redboxjs would be interested in? based on #302

**Comment by veeramarni** [comment]
@mjrussell Yes, we are using this in our project and will able to maintain it. Please add me as admin as I can add a few others to work on it.

**Comment by mjrussell** [comment]
Thanks @veeramarni! I've invited you but looks like you need to accept before I can give you any more access.

**Comment by goelshobhit** [comment]
Yes, I am using this in our project and will be able to maintain it.
